tutorials/3d/introduction_to_global_illumination.rst

```diff
@@ -0,0 +1,388 @@
+    but it can be done from both the editor and an exported project.
+
+- **SDFGI:** |average| Semi-real-time.
+
+  - Cascades are generated in real-time, making SDFGI
+    viable for procedurally generated levels (including when structures are generated
+    during gameplay).
+
+    Dynamic objects can *receive* GI, but not *contribute* to it. Emissive lighting
+    will only update when an object enters a cascade, so it may still work for
+    slow-moving objects.
+
+- **Screen-space indirect lighting (SSIL):** |good| Fully real-time.
+
+  - SSIL works with both static and dynamic lights. It also works with both
+    static and dynamic occluders (including emissive materials).
+
+- **LightmapGI:** |bad| Baked, and therefore not real-time.
+
+  - Both indirect lighting and SH reflections are baked and can't be changed at
+    run-time. Real-time GI must be
+    :ref:`simulated via other means <doc_faking_global_illumination>`,
+    such as real-time positional lights. Dynamic objects receive indirect lighting
+    via light probes, which can be placed automatically or manually by the user
+    (LightmapProbe node). Not viable for procedurally generated levels,
+    as baking lightmaps is only possible from the editor.
+
+- **ReflectionProbe:** |average| Optionally real-time.
+
+  - By default, reflections update when the probe is moved.
+    They update as often as possible if the update mode
+    is set to **Always** (which is expensive).
+
+  - Indirect lighting must be configured manually by the user, but can be changed
+    at run-time without causing an expensive computation to happen behind the scenes.
+    This makes ReflectionProbes viable for procedurally generated levels.
+
+User work needed
+^^^^^^^^^^^^^^^^
+
+- **VoxelGI:** |average| One or more VoxelGI nodes need to be created and baked.
+
+  - Adjusting extents correctly is required to get good results. Additionally
+    rotating the node and baking again can help combat leaks or streaking
+    artifacts in certain situations. Bake times are fast – usually below
+    10 seconds for a scene of medium complexity.
+
+- **SDFGI:** |good| Very little.
+
+  - SDFGI is fully automatic; it only needs to be enabled in the Environment resource.
+    The only manual work required is to set MeshInstances' bake mode property correctly.
+    No node needs to be created, and no baking is required.
+
+- **Screen-space indirect lighting (SSIL):** |good| Very little.
+
+  - SSIL is fully automatic; it only needs to be enabled in the Environment resource.
+    No node needs to be created, and no baking is required.
+
+- **LightmapGI:** |average| Requires UV2 setup and baking.
+
+  - Static meshes must be reimported with UV2 and lightmap generation enabled.
+    On a dedicated GPU, bake times are relatively fast thanks to the GPU-based
+    lightmap baking – usually below 1 minute for a scene of medium complexity.
+
+- **ReflectionProbe:** |bad| Placed manually by the user.
```

Comment:
As described in my last review, the user work needed is not accurately distilled into good|average|bad, only descriptions should be used
```suggestion
- **VoxelGI:** One or more VoxelGI nodes need to be created and baked.

  - Adjusting extents correctly is required to get good results. Additionally
    rotating the node and baking again can help combat leaks or streaking
    artifacts in certain situations. Bake times are fast – usually below
    10 seconds for a scene of medium complexity.

- **SDFGI:** Very little.

  - SDFGI is fully automatic; it only needs to be enabled in the Environment resource.
    The only manual work required is to set MeshInstances' bake mode property correctly.
    No node needs to be created, and no baking is required.

- **Screen-space indirect lighting (SSIL):** Very little.

  - SSIL is fully automatic; it only needs to be enabled in the Environment resource.
    No node needs to be created, and no baking is required.

- **LightmapGI:** Requires UV2 setup and baking.

  - Static meshes must be reimported with UV2 and lightmap generation enabled.
    On a dedicated GPU, bake times are relatively fast thanks to the GPU-based
    lightmap baking – usually below 1 minute for a scene of medium complexity.

- **ReflectionProbe:** Placed manually by the user.
```